mibs/fortinet/FORTINET-FORTIEXTENDER-MIB.mib

```diff
@@ -0,0 +1,7273 @@
+--
```

Comment:
please rename this file to `FORTINET-FORTIEXTENDER-MIB`

Reply:
My bad, uploaded the wrong version. Corrected!